src/features/defi/providers/fox-farming/components/FoxFarmingManager/Overview/WithdrawCard.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, Stack, useColorModeValue } from '@chakra-ui/react'
+import { Asset } from '@shapeshiftoss/asset-service'
+import {
+  DefiAction,
+  DefiParams,
+  DefiQueryParams,
+} from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import qs from 'qs'
+import { FaArrowDown, FaArrowRight } from 'react-icons/fa'
+import { Amount } from 'components/Amount/Amount'
+import { IconCircle } from 'components/IconCircle'
+import { Text } from 'components/Text'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useBrowserRouter } from 'hooks/useBrowserRouter/useBrowserRouter'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+
+type WithdrawCardProps = {
+  asset: Asset
+  amount: string
+}
+
+export const WithdrawCard = ({ asset, ...rest }: WithdrawCardProps) => {
```

Comment:
Can we change this over Fox LP and farming not to use ...restProps and destructure them directly at component-definition level?

Reply:
https://github.com/shapeshift/web/issues/2434